docs/romanisim/catalog.rst

```diff
@@ -31,6 +31,36 @@ Following these required fields is a series of columns giving the fluxes of the
 
 The simulator then renders these images in the scene and produces the simulated L1 or L2 images.
 
-The simulator API includes a few simple tools to generate parametric distributions of stars and galaxies.  The ``make_stars`` and ``make_galaxies`` routines make random catalogs of stars and galaxies.  The number of stars and galaxies can be adjusted.  Likewise, the power law index by which the sources' magnitudes are sampled can be adjusted, as can their limiting magnitudes.  Galaxy Sersic parameters, half-light radii, and position angles are chosen at random, with a rough attempt to make brighter galaxies appropriately larger (i.e., conserving surface brightness).  Stars can be chosen to be distributed with a King profile.  This functionality is however very rudimentary and limited, and is better suited for toy problems than real scientific work.  We expect scientific uses to be driven by custom-created catalogs rather than these simple routines.
+The simulator API includes a few simple tools to generate parametric distributions of stars and galaxies.  
+
+The ``make_stars`` and ``make_galaxies`` routines make random catalogs of stars and galaxies.  The number of stars and galaxies can be adjusted.  Likewise, the power law index by which the sources' magnitudes are sampled can be adjusted, as can their limiting magnitudes.  Galaxy Sersic parameters, half-light radii, and position angles are chosen at random, with a rough attempt to make brighter galaxies appropriately larger (i.e., conserving surface brightness).  Stars can be chosen to be distributed with a King profile.  This functionality is however very rudimentary and limited, and is better suited for toy problems than real scientific work.  
+
+We expect scientific uses to be driven by custom-created catalogs rather than the simple routines above, and provide the ``make_cosmos_galaxies`` and ``make_gaia_stars`` routines outlined below to provide that funcionality. 
+
+
+Using COSMOS Galaxies and GAIA Stars
+====================================
+
+.. figure:: ./cosmos_gaia.png
+  :width: 600
+
+  Level 3 Mosaic of cosmos and gaia sources with radius = 0.1 degree (search area visible)
+
+
+The simulator can utilize the COSMOS catalog to generate sources based off of real galaxies via the ``make_cosmos_galaxies`` method. The COSMOS survey is a 2 square degree collection of over 2 million galaxies with multi-wavelength photometry (Weaver et al. 2022)[https://arxiv.org/pdf/2110.13923]. A streamlined catalog for the simulator is provided with the code, but for those so interested, the latest full catalog can be found at https://cosmos.astro.caltech.edu/ and used with this method. 
+ 
+The simulator performs the following operations on the catalog sources:
+
+* Only galaxies that have "ultra-deep" Ultra Vista wavelength measurements are selected from the COSMOS catalog. This trims the catalog to 0.62 square degrees, and to a peak magnitude depth of ~25. 
```

Comment:
We should think about how to go fainter than 25th, which is not very deep for Roman.  e.g., this might look like:
- fit a power law to the number distribution as a function of magnitude for H < 24 or so.
- draw a number from that power law.
- If the number corresponds to H < 24, use the current approach.
- If the number corresponds to H > 24, use the current approach but then change all the magnitudes of the selected galaxy by a constant equal to H_drawn - H_selected galaxy.
That will screw up the colors as a function of redshift but will get large numbers of faint galaxies, which we'd like.  Maybe think about how much work such an implementation would be, and consider it as an option if it works?

Reply:
I was thinking about trying to base it off of the mag density of the HST observations and extrapolating from there. I think it is worth further discussion.

Reply:
Added this to the JIRA board